src/style.css

```diff
@@ -1 +1,76 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, serif;
+  font-weight: 300;
+}
+
+.form {
+  position: relative;
+  width: 100%;
+  margin: 20px 0;
+}
+
+.input {
+  outline: none;
+  font-family: inherit;
+  width: 100%;
+  border: 1px solid #e1e7ed;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)